### PR TITLE
[AIRFLOW-6402] SSH operator exception - NoneType object has no attribute startswith

### PR DESCRIPTION
**Pull request by ohadmata**
[AIRFLOW-6402]  https://issues.apache.org/jira/browse/AIRFLOW-6402

fix exception

### CODE REVIEW

File: airflow/utils/log/logging_mixin.py [hunk 1]
```diff
@@ -109,7 +109,8 @@ def _propagate_log(self, message):
         """
         Propagate message removing escape codes.
         """
-        self.logger.log(self.level, remove_escape_codes(message))
+        # self.logger.log(self.level, remove_escape_codes(message))
```
Comment (potiuk) [inline airflow/utils/log/logging_mixin.py:112]: I guess it's an overlook?

File: airflow/www/templates/airflow/ti_log.html [hunk 2]
```diff
@@ -123,8 +123,31 @@ <h4>{{ title }}</h4>
             if(auto_tailing && checkAutoTailingCondition()) {
               var should_scroll = true
             }
+
```
Comment (potiuk) [inline airflow/www/templates/airflow/ti_log.html:126]: Unrelated change?

**Review by potiuk — CHANGES_REQUESTED** [review]
I think there are more commits than needed here? Also It would be great if you follow our new beautiful shiny template we have when creating the PR.

**Comment by ohadmata** [comment]
I made a mistake here. sorry. will fix that in another pull request
